pom.xml

```diff
@@ -38,6 +43,19 @@
         <artifactId>slf4j-log4j12</artifactId>
         <version>1.7.21</version>
       </dependency>
+      <!-- Testing -->
+      <dependency>
+        <groupId>junit</groupId>
+        <artifactId>junit</artifactId>
+        <version>${junit.version}</version>
+        <scope>test</scope>
+      </dependency>
+      <dependency>
+        <groupId>org.mockito</groupId>
+        <artifactId>mockito-core</artifactId>
+        <version>${mockito-core.version}</version>
```

Comment:
IMO it's a bit redundant nevertheless since they are used only in one place each